app/lib/actions/remote_execution/run_host_job.rb

```diff
@@ -173,6 +194,22 @@ def determine_proxy!(proxy_selector, provider, host)
         end
         proxy
       end
+
+      extend ApipieDSL::Class
+      apipie :class, "An action representing execution of a job against a host" do
+        name 'Actions::RemoteExecution::RunHostJob'
+        refs 'Actions::RemoteExecution::RunHostJob'
+        sections only: %w[webhooks]
+        property :task, object_of: 'Task', desc: 'Returns the task to which this action belongs'
+        property :host_name, String, desc: "Returns the name of the host"
+        property :host_id, Integer, desc: "Returns the id of the host"
+        property :host, object_of: 'Host', desc: "Returns the host"
+        property :job_invocation_id, Integer, desc: "Returns the id of the job invocation"
+        property :job_invocation, object_of: 'JobInvocation', desc: "Returns the job invocation"
+      end
+      class Jail < ::Actions::ObservableAction::Jail
+        allow :host_name, :host_id, :host, :job_invocation_id, :job_invocation
+      end
```

Comment:
@ofedoren could you please take a look?

Reply:
Sure, forgot to mention that I generated docs with this PR as well and it seemed OK.

No concerns for now. I finally managed to configure my setup, I'll do some checks with this event occured and I'll try to refer the docs to get some info as a user would. If something will not correspond, I'll create a PR with fix afterwards.